app/controllers/page_views_controller.rb

```diff
@@ -4,74 +4,31 @@ class PageViewsController < ApplicationMetalController
   include ActionController::Head
 
   def create
-    page_view_create_params = if session_current_user_id
-                                page_view_params.merge(user_id: session_current_user_id)
-                              else
-                                page_view_params.merge(counts_for_number_of_views: 10)
-                              end
-
-    PageView.create(page_view_create_params)
+    page_view_create_params = params.slice(:article_id, :referrer, :user_agent)
+    if session_current_user_id
+      page_view_create_params[:user_id] = session_current_user_id
+    else
+      page_view_create_params[:counts_for_number_of_views] = 10
+    end
 
-    update_article_page_views
+    Articles::UpdatePageViewsWorker.perform_at(
+      5.minutes.from_now,
```

Comment:
I picked this value based on @mstruve's [comment](https://github.com/forem/forem/issues/8425#issuecomment-772117844), happy to update it if people prefer different values.

Reply:
I'm thinking out loud from the perspective of the consumer and 5 minutes to wait in the medium priority queue, for stats to update might be long. Maybe 2? Keeping the same ratio in the organic update. We can always change it if it leads to too much work for the server

Reply:
The only reason I can think of to try and keep the number a bit higher is because it increases the chances of sharing the execution of one worker between with multiple views on an article (smidge better performance).

But I can consider that an early optimization, so I'd be okay with 2 min as well. Once this is deployed we can see if adjusting it makes a difference or if it's even necessary to adjust at all.